src/krisi/evaluate/library/metric_wrappers.py

```diff
@@ -124,4 +126,4 @@ def pred_y_imbalance_ratio(
             f"pred_y_imbalance_ratio only works on binary classification problems, got more than 2 labels for either preds: {prediction_frequencies} or target: {target_frequencies}"
         )
 
-    return prediction_frequencies[pos_label] / target_frequencies[pos_label]
+    return abs(target_frequencies[pos_label] - prediction_frequencies[pos_label])
```

Comment:
the absolute difference of the two is interpretable

Reply:
I don't entirely understand, I thought we're substracting the ratios, but maybe not?

Reply:
so the absolute values should not matter

Reply:
@szemyd : `y_label_imbalance_ratio` is now always between 0-1, so `pred_y_imbalance_ratio` is interpretable in the larger context, with `y_label_imbalance_ratio`